release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -658,7 +658,9 @@ For more information, see xref:../networking/multiple_networks/assigning-a-secon
 [id="ocp-4-7-dev-exp"]
 === Developer experience
 
-
+==== Red Hat Openshift GitOps (Technology Preview)
+The Red Hat Openshift GitOps 1.0 Technology Preview release introduces a declarative way to implement continuous deployment for cloud native applications. You can use Red Hat OpenShift GitOps to adopt GitOps principles for managing cluster configurations, and automating secure, predictable, traceable, and repeatable application delivery across hybrid multi-cluster Kubernetes environments.
+It uses Argo CD as the core, and adds other tooling to enable teams to implement GitOps workflows across clusters.
```

Comment:
Looks good to me, however i request somebody from dev team please have a look. 

Ping @wtam2018 @bigkevmcd @chetan-rns